cmd/gafFilter/gafFilter.go

```diff
@@ -0,0 +1,76 @@
+// Command Group:
```

Comment:
Should we pick a command group for this program?

Reply:
if you run the gonomics command it will print out all the groups